dtfs-central-api/src/v1/validation/utilisation-report-service/utilisation-data-validator.js

```diff
@@ -0,0 +1,133 @@
+const validator = require('validator');
+const REGEXES = require('../../../constants/regex');
+
+/**
+ * Validates the UKEF ID to match the UKEF facility ID regex, returns an error message or null if valid.
+ * @param {Integer | String | null} ukefId - ukef facility ID.
```

Comment:
For all the validators, we could have the value we're checking as `unknown`. This better represents what we know about the value, and should also prompt us to check for the required type before calling any functions on it.